### PR TITLE
Operational testing

### PR DESCRIPTION
**Pull request by aburrell**
# Description

Adds tests for older (Python 3.6) systems.  Also addresses #109 by adding a cron job to run early on Mondays.

# Type of change

Please delete options that are not relevant.

- New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

On GitHub Actions.

# Checklist:

- [x] Make sure you are merging into the ``develop`` (not ``main``) branch
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Add a note to ``CHANGELOG.md``, summarizing the changes

If this is a release PR, replace the first item of the above checklist with the
release checklist on the pysat wiki:
https://github.com/pysat/pysat/wiki/Checklist-for-Release

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,13 +6,14 @@ This project adheres to [Semantic Versioning](https://semver.org/).
...
+  * Added tests for Python 3.6.8, continuing support for older systems
+  * Added an instrument for the LASP MgII core-to-wing index
+  * Added functions for general LISIRD downloads
 
 [0.0.8] - 2022-11-29
 --------------------
 * Bugs
-   * Fixed F10.7 prelim and daily metadata to allow the fill value to keep the
+  * Fixed F10.7 prelim and daily metadata to allow the fill value to keep the
      same type as the data
```
Comment (jklenzing) [inline CHANGELOG.md:17]:
```suggestion
  * Fixed F10.7 prelim and daily metadata to allow the fill value to keep the
    same type as the data
```

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -4,7 +4,11 @@
 
 name: Pytest with Flake8
 
-on: [push, pull_request]
+on:
+  pull_request:
+  push:
+  schedule:
+    - cron: "0 3 * * 1"
```
Comment (jklenzing) [inline .github/workflows/main.yml:11]: maybe add a comment so we remember this is for mondays

Comment (aburrell) [inline .github/workflows/main.yml:11]:
```suggestion
    - cron: "0 3 * * 1"  # Runs 03:00 UT on Mondays 
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,14 +6,15 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 [0.0.9] - 2023-01-18
 --------------------
 * Enhancements
-   * Added an instrument for the LASP MgII core-to-wing index
-   * Added functions for general LISIRD downloads
+  * Added tests for Python 3.6.8, continuing support for older systems
```
Comment (aburrell) [inline CHANGELOG.md:9]:
```suggestion
  * Added tests for Python 3.6.8, continuing support for older systems
  * Added a cron job for testing
```

**Review by jklenzing — APPROVED** [review]
Looks good!

**Pull request merged**